First pass for making did:pkh explicit in docs

bumblefudge opened the pull request.

bumblefudge reviewed (commented):
Thank to Tao and Daniel at Circle for all the input thus far!

bumblefudge commented:
diagram is here for future reference: https://swimlanes.io/d/ZQXgzSfos

dlim-circle commented on packages/docs/verite/patterns/identifier.md:
```diff
@@ -4,15 +4,13 @@ sidebar_position: 4
+The Verite sample implementations use [`did:key`](https://w3c-ccg.github.io/did-method-key/) (a scheme for representing offline/offchain key material in a DID-compatible format based on [the `multicodec` registry](https://github.com/multiformats/multicodec/blob/master/table.csv)), and [`did:web`](https://w3c-ccg.github.io/did-method-web/) (a simplifed resolution mechanism for using a well-known D ...
 
-On the subject/holder side, we chose did:key because it requires no fees or blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups. This enables identity wallet applications to mimic the behavior of existing crypto wallets.
+On the subject/holder side, we chose to model holder-side cryptography in the samepl implementation using `did:key` because it requires no gas/onchain fees nor assumes any blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups (i.e., the DID documents can be derived from the DID itself deterministically). This enables identity wallet ap ...
```
typo on `sample`

bumblefudge replied:
committed

dlim-circle commented on packages/docs/verite/patterns/smart-contract-verite.md:
```diff
@@ -35,11 +37,11 @@ Upon completing successful verification of Verifiable Credentials, verifiers cre
 - The expiration time of the result (which may occur sooner than the expiration of the credential)
 
-A Verification Result created by a verifier is never persisted to a blockchain.
+A Verite `Verification Result` created by a verifier should never be persisted directly to a blockchain.
```
I wonder, how does this work with persisting these info for audit trail ? Should those use some sort of IPFS instead ? 

Is the rationale behind this the privacy concern or are there other reasons ?

kimdhamilton replied:
The existing sentence is probably confusing. It's intended to be more of a statement of fact. 

If stored on-chain, the `Verification Result` is converted to a `Verification Record`. In some areas of the docs, we likely talk informally about how the payloads in the Result can have more data than what is ultimately persisted on-chain, and the idea was privacy concerns.

However, in practice, what matters is the signed part, which the smart contract needs to validate.

This use case is not immediately covered by the reference impl and may just cause confusion.

Maybe something like: 
"A Verification Result created by a verifier isn't directly persisted to a blockchain, but it may be (if on-chain persistence is needed by the use case) persisted as a minimized Verification Record"

dlim-circle replied:
Thanks for the clarification, the updated statement does answer my initial question.

bumblefudge replied:
good edit.  committed.

kimdhamilton commented on packages/docs/verite/patterns/identifier.md:
```diff
@@ -4,15 +4,13 @@ sidebar_position: 4
+The Verite sample implementations use [`did:key`](https://w3c-ccg.github.io/did-method-key/) (a scheme for representing offline/offchain key material in a DID-compatible format based on [the `multicodec` registry](https://github.com/multiformats/multicodec/blob/master/table.csv)), and [`did:web`](https://w3c-ccg.github.io/did-method-web/) (a simplifed resolution mechanism for using a well-known D ...
 
-On the subject/holder side, we chose did:key because it requires no fees or blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups. This enables identity wallet applications to mimic the behavior of existing crypto wallets.
+On the subject/holder side, we chose to model holder-side cryptography in the samepl implementation using `did:key` because it requires no gas/onchain fees nor assumes any blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups (i.e., the DID documents can be derived from the DID itself deterministically). This enables identity wallet ap ...
```
```suggestion
On the subject/holder side, we chose to model holder-side cryptography in the sample implementation using `did:key` because it requires no gas/onchain fees nor assumes any blockchain dependencies. It's a purely generative method, enabling resolution offline, without any registry lookups (i.e., the DID documents can be derived from the DID itself deterministically). This enables identity wallet applications to mimic the behavior of existing crypto wallets.
```

kimdhamilton commented on packages/docs/verite/patterns/smart-contract-verite.md:
```diff
@@ -88,7 +90,7 @@ The registry provides external-scoped accessor methods to obtain VerifierInfo fo
 ## Verifier Submission Pattern
 
-Either a verifier or a subject may submit a Verification Result for validation. Which pattern is appropriate depends upon the use case in question.
+Either a verifier or a subject may submit a Verification Result to a dapp directly for validation. Which pattern is appropriate depends upon the use case in question.
```
In theory a verifier or subject _could_ send a VR directly to a smart contract, which performs the validation, although -- especially for subject-submission -- this would be pretty rare.

That's context for a dumb question: does colloquial use of "dapp" also allow the interpretation a no-front-end / all-back-end (smart contract) situation?

bumblefudge replied:
edited for clarity

kimdhamilton commented on packages/docs/verite/patterns/identifier.md:
```diff
@@ -26,3 +24,13 @@ While implementors may choose any method they like, some common factors include:
 
 In general, DID method options will be influenced by a broader set of criteria relevant to all roles in the VC ecosystem, a comprehensive consideration of which is available in the [DID Method Rubric](https://w3c.github.io/did-rubric).
+
+## Wallet-based versus Address-based Holder Identification Schemes
```
```suggestion
## Wallet-based versus Account-based Holder Identification Schemes
```

kimdhamilton replied:
Not sure about my suggestion. Would this replacement allow for more consistency with the other docs that refer to "account-based"? Ignore this if I'm missing something

kimdhamilton reviewed (approved):
Fantastic work, @bumblefudge! I just left a few minor suggestions.

Thanks for taking this on!

bumblefudge commented:
Updated Account-bound diagram here:
https://swimlanes.io/d/OibugA5nf

Added a verbose Wallet-bound diagram here for balance/equivalence:
https://swimlanes.io/d/Q5cqL-HLW
